templates/README.md

```diff
@@ -80,9 +80,11 @@ The `integrity` hash will automatically be filled out by the app.
 A configuration file to override default behaviour of the app.
 
 ```yaml
-fixedReleaser: <GITHUB_USERNAME>
```

Comment:
you could make it non-breaking if you really wanted by permitting either a github username or a username followed by an email address in `<>`
I dunno if it's true that we're the only users, but I think it's fine to breaking if that's easier, since bzlmod itself is still experimental/release candidate anyhow.

Reply:
Just going off what I see on the Bazel Central Registry, but you're right I don't know for sure. I think at this point I am okay with the breaking change since I think there are very few users and as you said it's still experimental.